app/components/access_panels/library_location_popover_component.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module AccessPanels
+  class LibraryLocationPopoverComponent < ViewComponent::Base
+    # location.mhld
+    def initialize(mhld)
+      @mhld = mhld
+      super
+    end
+
+    def render?
+      @mhld.present? && @mhld.any? { |mhld_item| mhld_item.library_has.present? }
+    end
+
+    def popover_content
+      sanitize "<div class='fw-bold mb-2'>Library has:</div><div> #{item_locations}</div>"
+    end
+
+    def item_locations
+      @mhld.map do |mhld_item|
```

Comment:
If we use `safe_join` here, can we get rid of the `sanitize` on 16?

Reply:
Changed to do so.